src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -49,12 +46,24 @@ public CommandResult execute() {
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
+            final Set<String> wordsInName = new HashSet<>(convertStringListToLowerCase(person.getName().getWordsInName()));
             if (!Collections.disjoint(wordsInName, keywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
-
+    /**
+     * Converts an array of string into its lower cased version
+     *
+     * @param toConvert source ArrayList<String)
+     * @return All string in array converted to lower cased
+     */
+    private static List<String> convertStringListToLowerCase (List<String> toConvert) {
```

Comment:
Just a minor comment but another alternative method name you could have used was convertStringsToLowerCase.